src/config/constants.js

```diff
@@ -1,3 +1,68 @@
 const GA4_CONTAINER_ID = "GTM-KD86CMZ";
 
-module.exports = { GA4_CONTAINER_ID };
+const routeInfo = [
+  {
+    id: 1,
+    path: "/",
+    pageTitle: "Home ",
+    taxonomyLevel1: "HomeTax1",
+    taxonomyLevel2: "HomeTax2",
+  },
+  {
+    id: 2,
+    path: "/service-description",
+    pageTitle: "Service Description ",
+    taxonomyLevel1: "ServiceTax1",
+    taxonomyLevel2: "ServiceTax2",
+  },
+  {
+    id: 3,
+    path: "/organisation-type",
+    pageTitle: "Organisation Type ",
+    taxonomyLevel1: "OrganisationTax1",
+    taxonomyLevel2: "OrganisationTax2",
+  },
+  {
+    id: 4,
+    path: "/choose-location",
+    pageTitle: "Choose Location",
+    taxonomyLevel1: "ChooseLocTax1",
+    taxonomyLevel2: "ChooseLocTax2",
+  },
+  {
+    id: 5,
+    path: "/help-request",
+    pageTitle: "Help Request",
+    taxonomyLevel1: "HelpReqTax1",
+    taxonomyLevel2: "HelpReqTax2",
+  },
+  {
+    id: 6,
+    path: "/validate-organisation-type",
+    pageTitle: "Organisation Type",
+    taxonomyLevel1: "ErrorOrgTax1",
+    taxonomyLevel2: "ErorOrgTax2",
+  },
+  {
+    id: 7,
+    path: "/validate-help-request",
+    pageTitle: "Help Request",
+    taxonomyLevel1: "ErrorHelpTax1",
+    taxonomyLevel2: "ErrorHelpTax2",
+  },
+  {
+    id: 8,
+    path: "/validate-service-description",
+    pageTitle: "Service Description",
+    taxonomyLevel1: "ErrorSerivceTax1",
+    taxonomyLevel2: "ErrorServiceTax2",
+  },
+  {
+    id: 9,
+    path: "/validate-choose-location",
+    pageTitle: "Choose Location",
+    taxonomyLevel1: "ErrorChooseLocation1",
+    taxonomyLevel2: "ErrorChooseLocation2",
+  },
+];
+module.exports = { GA4_CONTAINER_ID, routeInfo };
```

Comment:
If you decide to use CAPS for your constant variable, you need to do it for all. Consistency is the key.
So routeInfo needs to be ROUTE_INFO

Reply:
changed to all caps